src/profile/components/ProfileDesktop/ProfileDesktop.tsx

```diff
@@ -0,0 +1,25 @@
+import { Copy, Spacer, Flex, Stack } from "@/shared/components"
+import { OrganizationType, SwitchOrganization, ChangePassword } from "@/profile"
+
+interface IProfileDesktopProps {
+  setUserOrganization: Function
+  userOrganization: OrganizationType | string
+}
+
+export const ProfileDesktop = ({ setUserOrganization, userOrganization }: IProfileDesktopProps) => {
+  return (
+    <Flex direction="column" css={{ maxWidth: "896px" }}>
```

Comment:
replace Flex with just <> and add comment below `{/* TODO: it should be Title, so need to fix it later */}`

Reply:
leave flex as we need to  add fix width for block